packages/metrics/src/config/EnvironmentVariablesService.ts

```diff
@@ -13,12 +13,23 @@ class EnvironmentVariablesService
 {
   private namespaceVariable = 'POWERTOOLS_METRICS_NAMESPACE';
 
+  private disabledVariable = 'POWERTOOLS_METRICS_DISABLED';
+
   /**
    * Get the value of the `POWERTOOLS_METRICS_NAMESPACE` environment variable.
    */
   public getNamespace(): string {
     return this.get(this.namespaceVariable);
   }
+
+  /**
+   * Get the value of the `POWERTOOLS_METRICS_DISABLED` environment variable.
+   */
+  public getMetricsDisabled(): boolean {
```

Comment:
Naming alternatives: isDisabled, getDisabled

Reply:
I think this is ok as it is.